packages/version/src/__tests__/version-bump.spec.ts

```diff
@@ -131,7 +131,11 @@ describe('version bump', () => {
     await factory(createArgv(testDir, '--bump', 'prerelease', '--scope', 'package-1'));
 
     const message = await getCommitMessage(testDir);
-    expect(message).toContain('package-1@1.0.1-alpha.0');
```

Comment:
ahh sorry could you also add this one back? I meant to have both the filtered package and the ones that are outside of your scope. So `contain(pkg-1)` and all other ones as `not.contain(...)`